uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp

```diff
@@ -25,65 +25,96 @@
 
     <!-- Portlet Titlebar -->
     <div class="fl-widget-titlebar titlebar portlet-titlebar" role="sectionhead">
-        <h2 class="title" role="heading"><spring:message code="delete.portlet.entities"/></h2>
+        <h2 class="title" role="heading">
+            <spring:message code="delete.portlet.entities"/>
+        </h2>
         <div class="toolbar" role="toolbar">
             <ul class="btn-group">
-                <li class="btn"><a class="button btn btn-primary" href="<portlet:renderURL/>"><spring:message code="import"/> <i class="fa fa-upload"></i></a></li>
-                <li class="btn"><a class="button btn btn-primary" href="<portlet:renderURL><portlet:param name="action" value="export"/></portlet:renderURL>"><spring:message code="export"/> <i class="fa fa-download"></i></a></li>
+                <li class="btn">
+                    <a class="button btn btn-primary" href="<portlet:renderURL/>">
+                        <spring:message code="import"/>
+                        <i class="fa fa-upload"></i>
+                    </a>
+                </li>
+                <li class="btn">
+                    <a class="button btn btn-primary" href="<portlet:renderURL><portlet:param name="action" value="export"/></portlet:renderURL>">
+                        <spring:message code="export"/>
+                        <i class="fa fa-download"></i>
+                    </a>
+                </li>
             </ul>
         </div>
     </div>
-    
+
     <!-- Portlet Content -->
-    <div class="fl-widget-content content portlet-content" role="main">   
-        
+    <div class="fl-widget-content content portlet-content" role="main">
+
         <!-- Messages -->
         <div class="portlet-msg-error portlet-msg error alert alert-danger" role="alert">
             <div class="titlebar">
-                <h3 class="title">Warning</h3>
+                <h3 class="title">
+                    Warning
+                </h3>
             </div>
             <div class="content">
-                <p>Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.</p>
+                <p>
+                    Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.
```

Comment:
:gb:
delete.portlet.warning.explanations=Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.
:fr:
delete.portlet.warning.explanations=Supprimer des entit\u00E9s peut faire des choses tellement vilaines \u00E0 votre portail.  Par d\u00E9faut, toutes les op\u00E9rations de suppression sont d\u00E9sactiv\u00E9es;  utilisez cette fonctionnalit\u00E9 avec grande pr\u00E9caution.